Extras/RationalResourcesNuclearFamily/RR_NuclearFamily_CRP_ThermalNozzles.cfg

```diff
@@ -621,6 +629,7 @@
 	
 	@MODULE[ModuleEnginesFX],1
 	{
+		%engineID = LqdCO2
```

Comment:
Are you sure this line is needed? Parts aren't cloned and split for thermal nozzles as they are for classic NTRs. Rather, I don't see a segment in here that ensures the air-breathing mode has a consistent engineID value in it.

Reply:
I mostly included it to keep it somewhat consistent with the NTR config, however I did forget to reflect those changes onto the air-breathing mode